index.js

```diff
@@ -1,7 +1,24 @@
 const GraphemeSplitter = require(`grapheme-splitter`);
```

Comment:
This could become a lazy require now, no need to load and evaluate `grapheme-splitter` when `Intl.Segmenter` is available

Reply:
Done :+1: